From d02010b1a664740081bc99fc910ba1bae95dc009 Mon Sep 17 00:00:00 2001
From: Ofer Heifetz <oferh@marvell.com>
Date: Sun, 19 Mar 2017 17:00:32 +0200
Subject: [PATCH 1503/2241] crypto: inside-secure: align the cipher and hash
 send routine rdesc count

nr_dst is the destination based on the sg_nents_for_len,
for_each_sg(nr_dst) we build a result descriptor and n_rdesc
is the counter for that accumulated result descriptors.

So in essence the nr_dst should be equal to n_rdesc,
but since the same routine for hash uses the n_rdesc,
We aligne the cipher code to use the n_rdesc instead of nr_dst.

No functional change, only readability.

Change-Id: Ie1dc1f84961dcced039654ec3b4b500c4f2c2163
Signed-off-by: Ofer Heifetz <oferh@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/37737
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
---
 drivers/crypto/inside-secure/cipher.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/crypto/inside-secure/cipher.c b/drivers/crypto/inside-secure/cipher.c
index 5fda6c2..94050a1 100644
--- a/drivers/crypto/inside-secure/cipher.c
+++ b/drivers/crypto/inside-secure/cipher.c
@@ -266,7 +266,7 @@ static int safexcel_aes_send(struct crypto_async_request *async,
 	spin_unlock_bh(&priv->ring[ring].egress_lock);
 
 	*commands = n_cdesc;
-	*results = nr_dst;
+	*results = n_rdesc;
 	return 0;
 
 rdesc_rollback:
-- 
2.7.4

